bin/themis.bat

```diff
@@ -17,7 +17,12 @@ if not exist "%STARTUP_SCRIPT%" (
 )
 
 rem FIXME: Some wrong case exists in passing the argument list.
-%THEMIS_VIM% -u NONE -i NONE -n -N %THEMIS_ARGS% --cmd "source %STARTUP_SCRIPT%" -- %* 2>&1
+rem DO NOT directly output the result to command prompt while 'normal' command in Vim script
+rem will move a cursor of command prompt in that case.
+set THEMIS_LOG="%USERPROFILE%\AppData\Local\Temp\themis.log"

```

Comment:
I'm not Windows user so not sure but it seems the path is used as a user's temporary directory so I thought it fit more than `%TMP%`

http://freesoft.tvbok.com/tips/optimise_vista/delete_temp_files.html